app/upgrades/v2_5_0/v2_5_0_test.go

```diff
@@ -188,56 +183,22 @@ func (s *Suite) TestUpgrade() {
 		s.Require().NoError(err)
 
 		ownerVal := new(struct{ Value gethcommon.Address })
-		err = compiledContract.ABI.UnpackIntoInterface(
-			ownerVal, methodName, evmResp.Ret,
-		)
+		err = compiledContract.ABI.UnpackIntoInterface(ownerVal, "owner", evmResp.Ret)
 		s.Require().NoError(err)
 		s.Require().Equal(evm.EVM_MODULE_ADDRESS.Hex(), ownerVal.Value.Hex())
 	})
 
 	s.Run("Confirm stNIBI ERC20 contract has new holder balances unharmed", func() {
 		// It MUST still be a contract
-		sdbAccForERC20 := deps.EvmKeeper.GetAccount(deps.Ctx, funtoken.Erc20Addr.Address)
-		s.Require().True(sdbAccForERC20.IsContract())
+		s.Require().True(deps.EvmKeeper.GetAccount(deps.Ctx, funtoken.Erc20Addr.Address).IsContract())
 
 		evmObj, _ := deps.NewEVMLessVerboseLogger()
-		for holderAddr, wantBal := range mainnetHolderBals {
-			balErc20, err := deps.EvmKeeper.ERC20().BalanceOf(
-				funtoken.Erc20Addr.Address, holderAddr, deps.Ctx, evmObj)
+		for idx, holderAddr := range holders {
+			balErc20, err := deps.EvmKeeper.ERC20().BalanceOf(funtoken.Erc20Addr.Address, holderAddr, deps.Ctx, evmObj)
 			s.Require().NoError(err)
-			s.Require().Equalf(wantBal.String(), balErc20.String(),
-				"holderAddr %s", holderAddr)
+			s.Require().Equal(strconv.Itoa(20*(idx+1)), balErc20.String())
```

Comment:
Here's the most important check. It validates that every holder's original balance is still in tact after the upgrade.